cluster/images/etcd-empty-dir-cleanup/etcd-empty-dir-cleanup.sh

```diff
@@ -17,7 +17,7 @@
 echo "Removing empty directories from etcd..."
 
 cleanup_empty_dirs () {
-  if [[ $(${ETCDCTL} ls $1) ]]; then
+  if [[ "$(${ETCDCTL} ls $1)" ]]; then
```

Comment:
I think `busybox`'s `ash` is faking `[[..]]` support.

Simple example inside the busybox container:
```console
# cat test.txt
foo
bar
# if [[ $(cat test.txt) ]]; then echo a; else echo b; fi
sh: bar: unknown operand
b
# if [[ "$(cat test.txt)" ]]; then echo a; else echo b; fi
a
```

I could probably change this to `[..]` if we want to be clearer that this isn't bash.